apps/emails/tests/test_helpers.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+import pytest
+import pytz
+
+from apps.emails.helpers import convert_to_timezone_formatted
+
+
+@pytest.mark.parametrize(
+    "timezone, fmt, isUTC, expect",
+    [
+        ("America/New_York", "%d/%m/%Y %H:%M %Z", False, "05/12/2022 19:00 EST"),
+        ("America/New_York", None, False, "12-05-22 19:00 EST"),
+        ("America/New_York", None, True, "12-05-22 19:00 EST"),
+        (None, None, False, "12-06-22 00:00 UTC"),
+    ],
+)
+def test__convert_to_timezone_formatted(timezone, fmt, isUTC, expect):
```

Comment:
This is minor, but can we get rid of `__` in favor of `_`. We're using `_` consistently elsewhere I believe, and would like to keep consistent.

Also, should use snake_case for var names.

```suggestion
def test_convert_to_timezone_formatted(timezone, fmt, is_UTC, expect):
```